app/src/components/main/navigationTabs/navigationTabs.jsx

```diff
@@ -51,6 +51,25 @@ export class NavigationTabs extends Component {
     customBlock: null,
   };
 
+  componentDidMount() {
+    const { activeTab, config } = this.props;
```

Comment:
I think will be good to move this check and action into separate function

Reply:
done